src/scripts/components/LoginPage.tsx

```diff
@@ -194,6 +215,14 @@ function SignupButton ({ isPopup }: Pick<LoginProps, 'isPopup'>) {
 
 const openPage = (url: string) => () => browser.tabs.create({ url });
 
+const performLogin = (username: string, password: string) => (
+  sendMessage({ type: 'login', username, password })
+);
+
+const onLoginSuccess = () => {
+  location.href = 'login.html?source=quick-start';
```

Comment:
when using the secondary login, we seem to do both of these:
- Direct to the quick-start page
- Open the settings page in a new tab

we should only do the first one?

Reply:
I couldn't reproduce this,
logging in opened the quick-start page for me
https://toggl.slack.com/files/U8U0D02S3/FRSDW6QRK/screen_recording_2019-12-17_at_12.14.31_pm.mov